ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0110__service_mesh.robot

```diff
@@ -58,15 +61,135 @@ Validate Service Mesh State Removed
     [Teardown]    Set Service Mesh State To Managed And Wait For CR Ready
     ...           ${SERVICE_MESH_CR_NAME}    ${SERVICE_MESH_CR_NS}    ${OPERATOR_NS}
 
+Validate Service Mesh Control Plane Already Created
+    [Documentation]    This Test Case validates that only one ServiceMeshControlPlane is allowed to be installed per project/namespace
+    [Tags]      Operator        Tier3       RHOAIENG-2517
+    Fetch Image Url And Update Channel
+    Check Whether DSC Exists And Save Component Statuses
+    Fetch Cluster Type By Domain
+    IF    "${CLUSTER_TYPE}" == "selfmanaged"
+        Uninstall RHODS In Self Managed Cluster
+        Create Smcp From Template
+        Install RHODS In Self Managed Cluster Using CLI     ${CLUSTER_TYPE}     ${IMAGE_URL}
+    ELSE IF    "${CLUSTER_TYPE}" == "managed"
+        Uninstall RHODS In OSD
+        Create Smcp From Template
+        Install RHODS In Managed Cluster Using CLI      ${CLUSTER_TYPE}     ${IMAGE_URL}
+    END
+    Operator Deployment Should Be Ready
+    # Go check the Operator logs for the error message: denied the request: only one service mesh may be installed per project/namespace
+    Verify Pod Logs Contain Error
+    [Teardown]      Teardown Service Mesh Control Plane Already Created
+
 
 *** Keywords ***
 Suite Setup
     [Documentation]    Suite Setup
     RHOSi Setup
     Wait Until Operator Ready    ${SERVICE_MESH_OPERATOR_DEPLOYMENT_NAME}    ${SERVICE_MESH_OPERATOR_NS}
-    Wait Until Operator Ready    ${OPERATOR_DEPLOYMENT_NAME}    ${OPERATOR_NS}
-    Wait For DSCI Ready State    ${DSCI_NAME}    ${OPERATOR_NS}
+    Wait Until Operator Ready    ${OPERATOR_DEPLOYMENT_NAME}    ${OPERATOR_NAMESPACE}
+    Wait For DSCI Ready State    ${DSCI_NAME}    ${OPERATOR_NAMESPACE}
 
 Suite Teardown
     [Documentation]    Suite Teardown
     RHOSi Teardown
+
+Teardown Service Mesh Control Plane Already Created
+    # Cleanup the SMCP
+    Delete Smcp
+    # Cleanup Olminstall dir
+    Cleanup Olm Install Dir
+    IF      ${DSC_EXISTS} == True
+        Apply DataScienceCluster CustomResource     ${DSC_NAME}     True    ${custom_cmp}
+    END
+
+Check Whether DSC Exists And Save Component Statuses
+    ${rc}=    Run And Return Rc
+    ...    oc get datasciencecluster ${DSC_NAME}
+    IF  ${rc} == 0
+        Set Global Variable    ${DSC_EXISTS}    True
+        &{custom_cmp} =       Create Dictionary
+        ${rc}    ${out}=    Run And Return Rc And Output
+        ...    oc get datasciencecluster ${DSC_NAME} -o jsonpath='{.spec.components}'
+        ${custom_cmp}=    evaluate    json.loads('''${out}''')    json
```

Comment:
try with `Load Json String` from `ods_ci/tests/Resources/Common.robot`

Reply:
This one is already working

Reply:
ok reusing Load Json String